src/fromager/packagesettings.py

```diff
@@ -628,6 +628,7 @@ class PackageBuildInfo:
     """
 
     def __init__(self, settings: Settings, ps: PackageSettings) -> None:
+        self._settings = settings
```

Comment:
I avoided to keep a reference to `settings` because it would introduce a circular dependency.

Reply:
Looks like the code never used the global settings. So removed it.